main.go

```diff
@@ -192,9 +192,10 @@ func runInTrainingMode(host string, reBaseAll bool, testSuite *testStrategies.Te
 
 	// Initialize the performance statistics struct.
 	perfStatsForTest := &perfTestUtils.PerfStats{
-		TestDate:             scenarioTimeStart,
```

Comment:
Renamed TestDate to TestTimeStart to be consistent with TestTimeEnd variable (see data structures)